src/LanguageServer/Impl/Definitions/ILogger.cs

```diff
@@ -19,5 +19,6 @@
 namespace Microsoft.Python.LanguageServer {
     public interface ILogger {
         void TraceMessage(IFormattable message);
+        void TraceMessage(string message);
```

Comment:
This has an issue of creating strings (lots of strings) even when logging is switched off or doesn't meet the required severity. It creates noticeable performance impact. If we need to get formatted strings from resources and can't use `IFormattable`, we can add `TraceMessageFormat(string format, params string[] parameters)`.

Reply:
OK